misc/Makefile

```diff
@@ -15,17 +17,45 @@ doc/qutebrowser.1.html:
 
 install: doc/qutebrowser.1.html
 	$(PYTHON) setup.py install --prefix="$(PREFIX)" --optimize=1 $(SETUPTOOLSOPTS)
+	$(PYTHON) setup.py install --mandir="$(datarootdir)/man" --optimize=1 $(SETUPTOOLSOPTS)
+	$(PYTHON) setup.py install --datadir="$(datarootdir)" --optimize=1 $(SETUPTOOLSOPTS)
 	install -Dm644 doc/qutebrowser.1 \
 		"$(DESTDIR)$(PREFIX)/share/man/man1/qutebrowser.1"
+	install -Dm644 doc/qutebrowser.1\
+	    "$(DESTDIR)$(datarootdir)/man/share/man/man1/qutebrowser.1"
+	install -Dm644 doc/qutebrowser.1\
+	    "$(DESTDIR)$(datarootdir)/share/man/man1/qutebrowser.1"
 	install -Dm644 misc/qutebrowser.desktop \
 		"$(DESTDIR)$(PREFIX)/share/applications/qutebrowser.desktop"
+	install -Dm644 misc/qutebrowser.desktop\
```

Comment:
I don't think the added lines after this are needed, we only want to put man files in the man directory (unless I'm misunderstanding this)